crates/uv-settings/src/lib.rs

```diff
@@ -187,6 +194,53 @@ fn config_dir() -> Option<PathBuf> {
     }
 }
 
+#[cfg(not(windows))]
+fn locate_system_config_xdg(value: Option<&str>) -> Option<PathBuf> {
+    // On Linux/MacOS systems, read the XDG_CONFIG_DIRS environment variable
+
+    let default = "/etc/xdg";
+    let config_dirs = value.filter(|s| !s.is_empty()).unwrap_or(default);
+
+    for dir in config_dirs.split(':').take_while(|s| !s.is_empty()) {
```

Comment:
Does `env::split_paths` work here?

Reply:
Empirically, appears not!

Reply:
Really? That's pretty surprising.

Reply:
Yeah. It returns `""` in the iterator, which then resolves to the current working directory.

Reply:
(We could filter empty string I guess.)

Reply:
Oh interesting. Yeah I guess it seems more correct to handle the empty string separately. I don't feel strongly since it's already platform-specific code though.